Source/WebKit/Shared/Cocoa/XPCEndpoint.h

```diff
@@ -27,16 +27,17 @@
 
 #ifdef __cplusplus
 
-#include "WKDeclarationSpecifiers.h"
+#include <WebKit/WKBase.h>
 #include <wtf/OSObjectPtr.h>
+
```

Comment:
I'm assuming these blank lines are intentional, or something check-webkit-style asked for?

Reply:
Not intentional, fixed in latest patch.

Thanks for reviewing!